managers.py

```diff
@@ -0,0 +1,48 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.connect(r"/cinema.sqlite")
```

Comment:
The database connection path is set to `/cinema.sqlite`. Ensure that this path is correct and accessible from your environment. If the database file is located elsewhere, update the path accordingly.